web/web.go

```diff
@@ -207,16 +207,20 @@ func New(o *Options) *Handler {
 		router.Get("/user/*filepath", instrf("user", route.FileServe(o.UserAssetsPath)))
 	}
 
-	if o.EnableQuit {
+	if o.EnableLifecycle {
 		router.Post("/-/quit", h.quit)
+		router.Post("/-/reload", h.reload)
+	} else {
+		router.Post("/-/quit", func(w http.ResponseWriter, _ *http.Request) {
+			w.WriteHeader(http.StatusForbidden)
+			w.Write([]byte("Lifecycle APIs are not enabled"))
```

Comment:
Even here it is passed in as a configuration option. We don't know that it was set through a flag.
I'd prefer to only do it if it becomes an understanding problem for users.